backend/backend/settings/dev.py

```diff
@@ -27,6 +27,8 @@
 SITE_HOST = f'substra-backend.{ORG_NAME}.xyz'
 SITE_PORT = DEFAULT_PORT
 DEFAULT_DOMAIN = os.environ.get('DEFAULT_DOMAIN', f'http://{SITE_HOST}:{SITE_PORT}')
+
+CELERY_RESULT_BACKEND = 'django-db'
```

Comment:
What's the purpose of putting this assignment in dev and prod settings instead of common?

Reply:
Because we use `common.py` for the celery beat.
Consequently, if so, as celerybeat is not connected to the postgresql db. when launching the periodic task, it will try to send the result to the django-db and fails !